4.LastFM/lastfmLoader.js

```diff
@@ -0,0 +1,21 @@
+function LastfmLoader(key, url) {
+  this.key = key || 'ea9abab35aef9b1a492f071c30b761b2';
+  this.url = url || 'http://ws.audioscrobbler.com/2.0/?format=json&';
+  this.query = '';
+  this.load = function(callback) {
+    var URL = this.url + 'api_key=' + this.key + '&' + this.query;
```

Comment:
I think it's possible to pass `url` argument without `?` like (null, 'try_me')
in this case URL is `try_meapi_key=`
I suggest to break url and query string